src/docker.ts

```diff
@@ -108,7 +113,7 @@ export default class Docker {
       }
 
       const vulnerabilities: Vulnerability[] = trivyJsonScanReport
-      if (vulnerabilities.length > 0) {
+      if (notifyTrivyAlert === 'true' && vulnerabilities.length > 0) {
```

Comment:
boolean 型にすれば、これで良さそうです。

```suggestion
      if (notifyTrivyAlert && vulnerabilities.length > 0) {
```